src/hooks/useDisputes.js

```diff
@@ -157,8 +157,14 @@ function useProcessedDispute(dispute) {
           return setDisputeProcessed({
             ...dispute,
             description: parsedDisputeData.description || disputeDescription,
-            agreementText: agreementText || '',
+            disputedActionText: parsedDisputeData.disputedActionText || '',
+            disputedActionURL: parsedDisputeData.disputedActionURL || '',
+            agreementText:
+              parsedDisputeData.agreementTitle || agreementText || '',
```

Comment:
Just found this a bit weird; should we place the agreement title in a different field? Would this always be replacing the agreement text now?

Reply:
Yeah, in this case the `agreementText` that comes from ipfs metadata is the relative path to the ipfs content (and `agreementUrl ` is the resolved link) and the agreementTitle is the description. 

Maybe we could ask for `agreementText` to be renamed to `agreementUrl` at the metadata level

Reply:
> Maybe we could ask for agreementText to be renamed to agreementUrl at the metadata level

Not sure on this—don't want to change it too much, but it just seems weird to set agreement**Title** for agreement**Text**.